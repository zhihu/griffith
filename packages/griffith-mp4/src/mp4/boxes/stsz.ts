import Stream from '../stream'

export default function stss(buffer: any) {
  const stream = new Stream(buffer)

  const version = stream.readByte(1)
  const flags = stream.readByte(3)

  const sampleSize = stream.readByte(4)
  const sampleCount = stream.readByte(4)
  const samples = []

  for (let i = 0; i < sampleCount; i++) {
    samples.push({entrySize: stream.readByte(4)})
  }

  const stssBox = {
    version,
    flags,
    sampleSize,
    samples,
  }

  return stssBox
}
