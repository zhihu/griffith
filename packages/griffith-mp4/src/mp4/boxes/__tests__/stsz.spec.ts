import stsz from '../stsz'

describe('stsz box', () => {
  it('should get the stsz box tree', () => {
    // prettier-ignore
    const stszBody = new Uint8Array([
      0x00, 0x00, 0x00, 0x00,  // sampleSize
      0x00, 0x00, 0x00, 0x00,  // sampleCount
      0x00, 0x00, 0x00, 0xFB,  // entrySize
      0x00, 0x00, 0x1D, 0x22,
      0x00, 0x00, 0x05, 0x9B,
      0x00, 0x00, 0x00, 0x25,
      0x00, 0x00, 0x00, 0x12,
      0x00, 0x00, 0x00, 0x26,
      0x00, 0x00, 0x00, 0x89,
      0x00, 0x00, 0x08, 0x25,
      0x00, 0x00, 0x03, 0x1A,
      0x00, 0x00, 0x06, 0x81,
      0x00, 0x00, 0x03, 0x41,
      0x00, 0x00, 0x00, 0x2F,
      0x00, 0x00, 0x06, 0x1B,
      0x00, 0x00, 0x03, 0x34,
      0x00, 0x00, 0x06, 0xF3,
      0x00, 0x00, 0x05, 0x7C,
      0x00, 0x00, 0x05, 0xF8,
      0x00, 0x00, 0x01, 0x9F,
      0x00, 0x00, 0x07, 0x2F,
      0x00, 0x00, 0x01, 0xC1,
      0x00, 0x00, 0x06, 0x95,
      0x00, 0x00, 0x01, 0xA3,
      0x00, 0x00, 0x05, 0x0C,
      0x00, 0x00, 0x05, 0xE6,
      0x00, 0x00, 0x01, 0x57,
      0x00, 0x00, 0x0B, 0x38,
      0x00, 0x00, 0x01, 0xEB,
      0x00, 0x00, 0x0C, 0x27,
      0x00, 0x00, 0x02, 0xF5,
      0x00, 0x00, 0x0B, 0xAE,
      0x00, 0x00, 0x03, 0xBC,
      0x00, 0x00, 0x0B, 0x5D,
      0x00, 0x00, 0x01, 0xE3,
      0x00, 0x00, 0x0B, 0x78,
      0x00, 0x00, 0x03, 0x51,
      0x00, 0x00, 0x09, 0x16,
      0x00, 0x00, 0x00, 0x5D,
      0x00, 0x00, 0x0C, 0xC3,
      0x00, 0x00, 0x02, 0x9E,
      0x00, 0x00, 0x0C, 0xE8,
      0x00, 0x00, 0x0B, 0x0E,
      0x00, 0x00, 0x03, 0xC7,
      0x00, 0x00, 0x01, 0x62,
      0x00, 0x00, 0x02, 0x38,
      0x00, 0x00, 0x09, 0x73,
      0x00, 0x00, 0x02, 0x28,
      0x00, 0x00, 0x01, 0xEC,
      0x00, 0x00, 0x0C, 0x50,
      0x00, 0x00, 0x08, 0x29,
      0x00, 0x00, 0x08, 0x15,
      0x00, 0x00, 0x08, 0x1E,
      0x00, 0x00, 0x09, 0x16,
      0x00, 0x00, 0x0F, 0x3D,
      0x00, 0x00, 0x03, 0x91,
      0x00, 0x00, 0x10, 0x0A,
      0x00, 0x00, 0x03, 0xB9,
      0x00, 0x00, 0x0D, 0x48,
      0x00, 0x00, 0x02, 0x97,
      0x00, 0x00, 0x01, 0xA2,
      0x00, 0x00, 0x0E, 0x27,
      0x00, 0x00, 0x05, 0x51,
      0x00, 0x00, 0x02, 0x9B,
      0x00, 0x00, 0x00, 0x52,
      0x00, 0x00, 0x10, 0x46,
      0x00, 0x00, 0x03, 0x91,
      0x00, 0x00, 0x08, 0xBE,
      0x00, 0x00, 0x01, 0xC6,
      0x00, 0x00, 0x08, 0x9E,
      0x00, 0x00, 0x0E, 0x7F,
      0x00, 0x00, 0x0B, 0x3C,
      0x00, 0x00, 0x0E, 0xDF,
      0x00, 0x00, 0x04, 0x00,
      0x00, 0x00, 0x0E, 0x09,
      0x00, 0x00, 0x0A, 0x32,
      0x00, 0x00, 0x03, 0xDA,
      0x00, 0x00, 0x02, 0x81,
      0x00, 0x00, 0x08, 0x4B,
      0x00, 0x00, 0x11, 0x7E,
      0x00, 0x00, 0x04, 0x4A,
      0x00, 0x00, 0x10, 0x8A,
      0x00, 0x00, 0x05, 0xD9,
      0x00, 0x00, 0x0D, 0xDA,
      0x00, 0x00, 0x04, 0x04,
      0x00, 0x00, 0x16, 0x03,
      0x00, 0x00, 0x07, 0x7A,
      0x00, 0x00, 0x0F, 0x3B,
      0x00, 0x00, 0x00, 0x90,
      0x00, 0x00, 0x0F, 0xEC,
      0x00, 0x00, 0x04, 0x3C,
      0x00, 0x00, 0x11, 0x7C,
      0x00, 0x00, 0x04, 0xC0,
      0x00, 0x00, 0x0B, 0x1B,
      0x00, 0x00, 0x03, 0x14,
      0x00, 0x00, 0x0C, 0x5A,
      0x00, 0x00, 0x04, 0x81,
      0x00, 0x00, 0x09, 0xC7,
      0x00, 0x00, 0x03, 0x7F,
      0x00, 0x00, 0x0C, 0x9F,
      0x00, 0x00, 0x05, 0xF3,
      0x00, 0x00, 0x0C, 0x46,
      0x00, 0x00, 0x06, 0x35,
      0x00, 0x00, 0x14, 0xF0,
      0x00, 0x00, 0x06, 0xD1,
      0x00, 0x00, 0x0A, 0xB5,
      0x00, 0x00, 0x02, 0x95,
      0x00, 0x00, 0x0C, 0x50,
      0x00, 0x00, 0x0E, 0x02,
      0x00, 0x00, 0x05, 0x01,
      0x00, 0x00, 0x0F, 0x20,
      0x00, 0x00, 0x05, 0xD5,
      0x00, 0x00, 0x0B, 0xE5,
      0x00, 0x00, 0x00, 0x63,
      0x00, 0x00, 0x0D, 0x62,
      0x00, 0x00, 0x04, 0xDE,
      0x00, 0x00, 0x0E, 0xAB,
      0x00, 0x00, 0x0D, 0x17,
      0x00, 0x00, 0x05, 0x07,
      0x00, 0x00, 0x08, 0x32,
      0x00, 0x00, 0x0D, 0xA2,
      0x00, 0x00, 0x0B, 0x8C,
      0x00, 0x00, 0x04, 0x0F,
      0x00, 0x00, 0x07, 0xB1,
      0x00, 0x00, 0x0C, 0x37,
      0x00, 0x00, 0x07, 0x26,
      0x00, 0x00, 0x07, 0xBC,
      0x00, 0x00, 0x0B, 0x34,
      0x00, 0x00, 0x14, 0xB9,
      0x00, 0x00, 0x0E, 0x9C,
      0x00, 0x00, 0x05, 0xED,
      0x00, 0x00, 0x10, 0x31,
      0x00, 0x00, 0x11, 0x2B,
      0x00, 0x00, 0x0F, 0x2D,
      0x00, 0x00, 0x02, 0xBE,
      0x00, 0x00, 0x08, 0x64,
      0x00, 0x00, 0x08, 0x90,
      0x00, 0x00, 0x0D, 0xFF,
      0x00, 0x00, 0x00, 0x84,
      0x00, 0x00, 0x06, 0xC8,
      0x00, 0x00, 0x07, 0x47,
      0x00, 0x00, 0x09, 0x44,
      0x00, 0x00, 0x01, 0xF8,
      0x00, 0x00, 0x08, 0x0C,
      0x00, 0x00, 0x0A, 0x2F,
      0x00, 0x00, 0x12, 0x22,
      0x00, 0x00, 0x02, 0xE1,
      0x00, 0x00, 0x08, 0x6E,
      0x00, 0x00, 0x01, 0xA9,
      0x00, 0x00, 0x0E, 0xAC,
      0x00, 0x00, 0x02, 0xDC,
      0x00, 0x00, 0x0A, 0xE2,
      0x00, 0x00, 0x01, 0x81,
      0x00, 0x00, 0x14, 0x7E,
      0x00, 0x00, 0x01, 0x7E,
      0x00, 0x00, 0x11, 0xB2,
      0x00, 0x00, 0x01, 0x67,
      0x00, 0x00, 0x01, 0x51,
      0x00, 0x00, 0x09, 0x8C,
      0x00, 0x00, 0x01, 0x69,
      0x00, 0x00, 0x00, 0xB8,
      0x00, 0x00, 0x01, 0x07,
      0x00, 0x00, 0x06, 0xF8,
      0x00, 0x00, 0x01, 0x30,
      0x00, 0x00, 0x00, 0x36,
      0x00, 0x00, 0x00, 0xC8,
      0x00, 0x00, 0x06, 0xC8,
      0x00, 0x00, 0x01, 0x08,
      0x00, 0x00, 0x00, 0xDB,
      0x00, 0x00, 0x00, 0x83,
      0x00, 0x00, 0x04, 0x8F,
      0x00, 0x00, 0x00, 0xB4,
      0x00, 0x00, 0x00, 0x50,
      0x00, 0x00, 0x00, 0x22,
      0x00, 0x00, 0x03, 0x5E,
      0x00, 0x00, 0x00, 0xDA,
      0x00, 0x00, 0x00, 0x4A,
      0x00, 0x00, 0x00, 0x86,
      0x00, 0x00, 0x0C, 0xC3,
      0x00, 0x00, 0x00, 0xC7,
      0x00, 0x00, 0x00, 0x22,
      0x00, 0x00, 0x03, 0x7D,
      0x00, 0x00, 0x00, 0xBA,
      0x00, 0x00, 0x00, 0x6D,
      0x00, 0x00, 0x00, 0x18,
      0x00, 0x00, 0x0C, 0x10,
      0x00, 0x00, 0x01, 0x01,
      0x00, 0x00, 0x02, 0x52,
      0x00, 0x00, 0x00, 0x53,
      0x00, 0x00, 0x0A, 0x44,
      0x00, 0x00, 0x08, 0x2A,
      0x00, 0x00, 0x0A, 0xF3,
      0x00, 0x00, 0x0B, 0xCB,
      0x00, 0x00, 0x03, 0x58,
      0x00, 0x00, 0x11, 0xB2,
      0x00, 0x00, 0x05, 0x7E,
      0x00, 0x00, 0x0A, 0xCF,
      0x00, 0x00, 0x0E, 0x81,
      0x00, 0x00, 0x05, 0x39,
      0x00, 0x00, 0x10, 0x45,
      0x00, 0x00, 0x0A, 0xC8,
      0x00, 0x00, 0x10, 0xC8,
      0x00, 0x00, 0x04, 0x42,
      0x00, 0x00, 0x14, 0x1C,
      0x00, 0x00, 0x05, 0x09,
      0x00, 0x00, 0x07, 0x93,
      0x00, 0x00, 0x09, 0x3F,
      0x00, 0x00, 0x0E, 0x3F,
      0x00, 0x00, 0x0B, 0xDF,
      0x00, 0x00, 0x02, 0xD6,
      0x00, 0x00, 0x0D, 0x54,
      0x00, 0x00, 0x12, 0xB8,
      0x00, 0x00, 0x04, 0x6C,
      0x00, 0x00, 0x00, 0x25,
      0x00, 0x00, 0x09, 0x08,
      0x00, 0x00, 0x02, 0x19,
      0x00, 0x00, 0x0D, 0x7F,
      0x00, 0x00, 0x03, 0xCE,
      0x00, 0x00, 0x0B, 0xB5,
      0x00, 0x00, 0x02, 0x6C,
      0x00, 0x00, 0x0B, 0xA9,
      0x00, 0x00, 0x03, 0xF2,
      0x00, 0x00, 0x02, 0xB6,
      0x00, 0x00, 0x01, 0xBE,
      0x00, 0x00, 0x09, 0x2E,
      0x00, 0x00, 0x04, 0xC6,
      0x00, 0x00, 0x02, 0xCE,
      0x00, 0x00, 0x02, 0x78,
      0x00, 0x00, 0x0C, 0x82,
      0x00, 0x00, 0x05, 0x23,
      0x00, 0x00, 0x0B, 0x1D,
      0x00, 0x00, 0x04, 0x9E,
      0x00, 0x00, 0x16, 0x74,
      0x00, 0x00, 0x0D, 0x8C,
      0x00, 0x00, 0x02, 0xAA,
      0x00, 0x00, 0x05, 0x93,
      0x00, 0x00, 0x0B, 0xB8,
      0x00, 0x00, 0x08, 0xF2,
      0x00, 0x00, 0x00, 0x4F,
      0x00, 0x00, 0x07, 0x93,
      0x00, 0x00, 0x06, 0xE8,
      0x00, 0x00, 0x09, 0xDA,
      0x00, 0x00, 0x05, 0xFC,
      0x00, 0x00, 0x06, 0xA7,
      0x00, 0x00, 0x05, 0x9B,
      0x00, 0x00, 0x08, 0xB7,
      0x00, 0x00, 0x07, 0xE1,
      0x00, 0x00, 0x07, 0xDE,
      0x00, 0x00, 0x06, 0xDE,
      0x00, 0x00, 0x07, 0x73,
      0x00, 0x00, 0x06, 0x91,
      0x00, 0x00, 0x06, 0xBF,
      0x00, 0x00, 0x05, 0x04,
      0x00, 0x00, 0x04, 0xF6,
    ])

    expect(stsz(stszBody)).toMatchSnapshot()
  })
})
