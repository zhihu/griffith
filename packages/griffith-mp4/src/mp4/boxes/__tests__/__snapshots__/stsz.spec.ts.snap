// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stsz box should get the stsz box tree 1`] = `
Object {
  "flags": 0,
  "sampleSize": 0,
  "samples": Array [
    Object {
      "entrySize": 7458,
    },
    Object {
      "entrySize": 1435,
    },
    Object {
      "entrySize": 37,
    },
    Object {
      "entrySize": 18,
    },
    Object {
      "entrySize": 38,
    },
    Object {
      "entrySize": 137,
    },
    Object {
      "entrySize": 2085,
    },
    Object {
      "entrySize": 794,
    },
    Object {
      "entrySize": 1665,
    },
    Object {
      "entrySize": 833,
    },
    Object {
      "entrySize": 47,
    },
    Object {
      "entrySize": 1563,
    },
    Object {
      "entrySize": 820,
    },
    Object {
      "entrySize": 1779,
    },
    Object {
      "entrySize": 1404,
    },
    Object {
      "entrySize": 1528,
    },
    Object {
      "entrySize": 415,
    },
    Object {
      "entrySize": 1839,
    },
    Object {
      "entrySize": 449,
    },
    Object {
      "entrySize": 1685,
    },
    Object {
      "entrySize": 419,
    },
    Object {
      "entrySize": 1292,
    },
    Object {
      "entrySize": 1510,
    },
    Object {
      "entrySize": 343,
    },
    Object {
      "entrySize": 2872,
    },
    Object {
      "entrySize": 491,
    },
    Object {
      "entrySize": 3111,
    },
    Object {
      "entrySize": 757,
    },
    Object {
      "entrySize": 2990,
    },
    Object {
      "entrySize": 956,
    },
    Object {
      "entrySize": 2909,
    },
    Object {
      "entrySize": 483,
    },
    Object {
      "entrySize": 2936,
    },
    Object {
      "entrySize": 849,
    },
    Object {
      "entrySize": 2326,
    },
    Object {
      "entrySize": 93,
    },
    Object {
      "entrySize": 3267,
    },
    Object {
      "entrySize": 670,
    },
    Object {
      "entrySize": 3304,
    },
    Object {
      "entrySize": 2830,
    },
    Object {
      "entrySize": 967,
    },
    Object {
      "entrySize": 354,
    },
    Object {
      "entrySize": 568,
    },
    Object {
      "entrySize": 2419,
    },
    Object {
      "entrySize": 552,
    },
    Object {
      "entrySize": 492,
    },
    Object {
      "entrySize": 3152,
    },
    Object {
      "entrySize": 2089,
    },
    Object {
      "entrySize": 2069,
    },
    Object {
      "entrySize": 2078,
    },
    Object {
      "entrySize": 2326,
    },
    Object {
      "entrySize": 3901,
    },
    Object {
      "entrySize": 913,
    },
    Object {
      "entrySize": 4106,
    },
    Object {
      "entrySize": 953,
    },
    Object {
      "entrySize": 3400,
    },
    Object {
      "entrySize": 663,
    },
    Object {
      "entrySize": 418,
    },
    Object {
      "entrySize": 3623,
    },
    Object {
      "entrySize": 1361,
    },
    Object {
      "entrySize": 667,
    },
    Object {
      "entrySize": 82,
    },
    Object {
      "entrySize": 4166,
    },
    Object {
      "entrySize": 913,
    },
    Object {
      "entrySize": 2238,
    },
    Object {
      "entrySize": 454,
    },
    Object {
      "entrySize": 2206,
    },
    Object {
      "entrySize": 3711,
    },
    Object {
      "entrySize": 2876,
    },
    Object {
      "entrySize": 3807,
    },
    Object {
      "entrySize": 1024,
    },
    Object {
      "entrySize": 3593,
    },
    Object {
      "entrySize": 2610,
    },
    Object {
      "entrySize": 986,
    },
    Object {
      "entrySize": 641,
    },
    Object {
      "entrySize": 2123,
    },
    Object {
      "entrySize": 4478,
    },
    Object {
      "entrySize": 1098,
    },
    Object {
      "entrySize": 4234,
    },
    Object {
      "entrySize": 1497,
    },
    Object {
      "entrySize": 3546,
    },
    Object {
      "entrySize": 1028,
    },
    Object {
      "entrySize": 5635,
    },
    Object {
      "entrySize": 1914,
    },
    Object {
      "entrySize": 3899,
    },
    Object {
      "entrySize": 144,
    },
    Object {
      "entrySize": 4076,
    },
    Object {
      "entrySize": 1084,
    },
    Object {
      "entrySize": 4476,
    },
    Object {
      "entrySize": 1216,
    },
    Object {
      "entrySize": 2843,
    },
    Object {
      "entrySize": 788,
    },
    Object {
      "entrySize": 3162,
    },
    Object {
      "entrySize": 1153,
    },
    Object {
      "entrySize": 2503,
    },
    Object {
      "entrySize": 895,
    },
    Object {
      "entrySize": 3231,
    },
    Object {
      "entrySize": 1523,
    },
    Object {
      "entrySize": 3142,
    },
    Object {
      "entrySize": 1589,
    },
    Object {
      "entrySize": 5360,
    },
    Object {
      "entrySize": 1745,
    },
    Object {
      "entrySize": 2741,
    },
    Object {
      "entrySize": 661,
    },
    Object {
      "entrySize": 3152,
    },
    Object {
      "entrySize": 3586,
    },
    Object {
      "entrySize": 1281,
    },
    Object {
      "entrySize": 3872,
    },
    Object {
      "entrySize": 1493,
    },
    Object {
      "entrySize": 3045,
    },
    Object {
      "entrySize": 99,
    },
    Object {
      "entrySize": 3426,
    },
    Object {
      "entrySize": 1246,
    },
    Object {
      "entrySize": 3755,
    },
    Object {
      "entrySize": 3351,
    },
    Object {
      "entrySize": 1287,
    },
    Object {
      "entrySize": 2098,
    },
    Object {
      "entrySize": 3490,
    },
    Object {
      "entrySize": 2956,
    },
    Object {
      "entrySize": 1039,
    },
    Object {
      "entrySize": 1969,
    },
    Object {
      "entrySize": 3127,
    },
    Object {
      "entrySize": 1830,
    },
    Object {
      "entrySize": 1980,
    },
    Object {
      "entrySize": 2868,
    },
    Object {
      "entrySize": 5305,
    },
    Object {
      "entrySize": 3740,
    },
    Object {
      "entrySize": 1517,
    },
    Object {
      "entrySize": 4145,
    },
    Object {
      "entrySize": 4395,
    },
    Object {
      "entrySize": 3885,
    },
    Object {
      "entrySize": 702,
    },
    Object {
      "entrySize": 2148,
    },
    Object {
      "entrySize": 2192,
    },
    Object {
      "entrySize": 3583,
    },
    Object {
      "entrySize": 132,
    },
    Object {
      "entrySize": 1736,
    },
    Object {
      "entrySize": 1863,
    },
    Object {
      "entrySize": 2372,
    },
    Object {
      "entrySize": 504,
    },
    Object {
      "entrySize": 2060,
    },
    Object {
      "entrySize": 2607,
    },
    Object {
      "entrySize": 4642,
    },
    Object {
      "entrySize": 737,
    },
    Object {
      "entrySize": 2158,
    },
    Object {
      "entrySize": 425,
    },
    Object {
      "entrySize": 3756,
    },
    Object {
      "entrySize": 732,
    },
    Object {
      "entrySize": 2786,
    },
    Object {
      "entrySize": 385,
    },
    Object {
      "entrySize": 5246,
    },
    Object {
      "entrySize": 382,
    },
    Object {
      "entrySize": 4530,
    },
    Object {
      "entrySize": 359,
    },
    Object {
      "entrySize": 337,
    },
    Object {
      "entrySize": 2444,
    },
    Object {
      "entrySize": 361,
    },
    Object {
      "entrySize": 184,
    },
    Object {
      "entrySize": 263,
    },
    Object {
      "entrySize": 1784,
    },
    Object {
      "entrySize": 304,
    },
    Object {
      "entrySize": 54,
    },
    Object {
      "entrySize": 200,
    },
    Object {
      "entrySize": 1736,
    },
    Object {
      "entrySize": 264,
    },
    Object {
      "entrySize": 219,
    },
    Object {
      "entrySize": 131,
    },
    Object {
      "entrySize": 1167,
    },
    Object {
      "entrySize": 180,
    },
    Object {
      "entrySize": 80,
    },
    Object {
      "entrySize": 34,
    },
    Object {
      "entrySize": 862,
    },
    Object {
      "entrySize": 218,
    },
    Object {
      "entrySize": 74,
    },
    Object {
      "entrySize": 134,
    },
    Object {
      "entrySize": 3267,
    },
    Object {
      "entrySize": 199,
    },
    Object {
      "entrySize": 34,
    },
    Object {
      "entrySize": 893,
    },
    Object {
      "entrySize": 186,
    },
    Object {
      "entrySize": 109,
    },
    Object {
      "entrySize": 24,
    },
    Object {
      "entrySize": 3088,
    },
    Object {
      "entrySize": 257,
    },
    Object {
      "entrySize": 594,
    },
    Object {
      "entrySize": 83,
    },
    Object {
      "entrySize": 2628,
    },
    Object {
      "entrySize": 2090,
    },
    Object {
      "entrySize": 2803,
    },
    Object {
      "entrySize": 3019,
    },
    Object {
      "entrySize": 856,
    },
    Object {
      "entrySize": 4530,
    },
    Object {
      "entrySize": 1406,
    },
    Object {
      "entrySize": 2767,
    },
    Object {
      "entrySize": 3713,
    },
    Object {
      "entrySize": 1337,
    },
    Object {
      "entrySize": 4165,
    },
    Object {
      "entrySize": 2760,
    },
    Object {
      "entrySize": 4296,
    },
    Object {
      "entrySize": 1090,
    },
    Object {
      "entrySize": 5148,
    },
    Object {
      "entrySize": 1289,
    },
    Object {
      "entrySize": 1939,
    },
    Object {
      "entrySize": 2367,
    },
    Object {
      "entrySize": 3647,
    },
    Object {
      "entrySize": 3039,
    },
    Object {
      "entrySize": 726,
    },
    Object {
      "entrySize": 3412,
    },
    Object {
      "entrySize": 4792,
    },
    Object {
      "entrySize": 1132,
    },
    Object {
      "entrySize": 37,
    },
    Object {
      "entrySize": 2312,
    },
    Object {
      "entrySize": 537,
    },
    Object {
      "entrySize": 3455,
    },
    Object {
      "entrySize": 974,
    },
    Object {
      "entrySize": 2997,
    },
    Object {
      "entrySize": 620,
    },
    Object {
      "entrySize": 2985,
    },
    Object {
      "entrySize": 1010,
    },
    Object {
      "entrySize": 694,
    },
    Object {
      "entrySize": 446,
    },
    Object {
      "entrySize": 2350,
    },
    Object {
      "entrySize": 1222,
    },
    Object {
      "entrySize": 718,
    },
    Object {
      "entrySize": 632,
    },
    Object {
      "entrySize": 3202,
    },
    Object {
      "entrySize": 1315,
    },
    Object {
      "entrySize": 2845,
    },
    Object {
      "entrySize": 1182,
    },
    Object {
      "entrySize": 5748,
    },
    Object {
      "entrySize": 3468,
    },
    Object {
      "entrySize": 682,
    },
    Object {
      "entrySize": 1427,
    },
    Object {
      "entrySize": 3000,
    },
    Object {
      "entrySize": 2290,
    },
    Object {
      "entrySize": 79,
    },
    Object {
      "entrySize": 1939,
    },
    Object {
      "entrySize": 1768,
    },
    Object {
      "entrySize": 2522,
    },
    Object {
      "entrySize": 1532,
    },
    Object {
      "entrySize": 1703,
    },
    Object {
      "entrySize": 1435,
    },
    Object {
      "entrySize": 2231,
    },
    Object {
      "entrySize": 2017,
    },
    Object {
      "entrySize": 2014,
    },
    Object {
      "entrySize": 1758,
    },
    Object {
      "entrySize": 1907,
    },
    Object {
      "entrySize": 1681,
    },
    Object {
      "entrySize": 1727,
    },
    Object {
      "entrySize": 1284,
    },
    Object {
      "entrySize": 1270,
    },
  ],
  "version": 0,
}
`;
